### ISSUE TITLE
Move Transition Not Showing Up in OBS

### ISSUE THREAD

**Issue by hayshmitty** [exeldro/obs-move-transition#194]
Hi there! I've downloaded the OBS plugin "Move Transition" manually and with the express installation. It shows in my Program Files and I've updated my visual c++ but it's still not showing up in OBS ....

Any solutions?

**Comment by exeldro** [comment]
Is it showing in your OBS log file? Does it show when you click the plus icon in the scene transitions dock?

**Comment by hayshmitty** [comment]
It's not showing up in my log file but I downloaded it according to the instructions and it shows up in my OBS Program Files

**Comment by exeldro** [comment]
is `move-transition.dll` in `C:\Program Files\obs-studio\obs-plugins\64bit` ?

**Comment by hayshmitty** [comment]
It does show that it's in that folder
![movetransition](https://github.com/exeldro/obs-move-transition/assets/158408509/d1211240-a7ff-4d65-a09c-aedd2e2c53b1)

**Comment by exeldro** [comment]
That looks like you have obs installed in not the default path, normally there should be a lot more files in that folder like for example `obs-outputs.dll`. Try to find the correct path by right clicking your obs icon and opening the properties.

**Comment by hayshmitty** [comment]
You're right I did end up looking in the default path and it does still show it's downloaded in there
![movetransition](https://github.com/exeldro/obs-move-transition/assets/158408509/e1bd6ab4-d5bc-4d64-afe5-956332455f27)

**Comment by exeldro** [comment]
The files look like a 3 year old version of OBS. Update OBS or use an old version of the move plugin.

**Comment by eweaz** [comment]
![image](https://github.com/exeldro/obs-move-transition/assets/158624916/cbcc3eda-5754-4eb6-b340-3b767932e6d3)
![image](https://github.com/exeldro/obs-move-transition/assets/158624916/6154adfb-a08f-4d86-8d3c-313f5bada2dd)
![image](https://github.com/exeldro/obs-move-transition/assets/158624916/42133fd8-8517-47b2-b2da-1e372da5cf15)
Dont show

**Comment by exeldro** [comment]
You should use the plus icon in the dock instead of checking in that list

**Comment by kamillou** [comment]
Hello, I have the same problem with the latest version of obs. I don't know what to do... :(

**Comment by exeldro** [comment]
@kamillou have you tried all steps mentioned above? and what are the results?

**Comment by kamillou** [comment]
yes ! I even tried the installer

**Comment by 174n** [comment]
I have the same problem. I'm using OBS Studio 30.1.2 (64 bit) on windows 10. I tried both installer and manual install.

In the logs it looks like it's loading without issues, but I don't see it in the scene transitions

```log
[Move Transition] loaded version 2.12.0
```

![image](https://github.com/exeldro/obs-move-transition/assets/13032812/e6995661-109c-47bf-b483-b2c16c78bef0)

**Comment by jertek03-png** [comment]
@exeldro 
i faced that issue too.
installed in correct path "C:\Program Files\obs-studio\obs-plugins\64bit". 
OBS version = 31.0.2
Sample of log file 
"17:39:06.098: Skipping module '../../obs-plugins/64bit/libEGL.dll', not an OBS plugin
17:39:06.098: Skipping module '../../obs-plugins/64bit/libGLESv2.dll', not an OBS plugin
**_17:39:06.099: Module '../../obs-plugins/64bit/move-transition.dll' not loaded_**
17:39:06.099: [NVIDIA Audio Effects:] NVIDIA denoiser disabled, redistributable not found or could not be loaded.
17:39:06.100: Failed to get NVVideoEffects.dll version info size
17:39:06.100: [NVIDIA VIDEO FX]: FX disabled, redistributable not found or could not be loaded.
17:39:06.108: [obs-browser]: Version 2.24.5"

**Comment by exeldro** [comment]
@jertek03-png the current version of the move plugin requires OBS 31.1.0 or higher

**Comment by jertek03-png** [comment]
@exeldro Thanks mate. After update the OBS, its working now
